_layouts/people.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "People Involved in the Conference"
+---
+
+<div style="margin-top:2em;">
+    <h2>Local Organizing Committee</h4>
+    <ul>
+    {% for omember in page.organizers %}
+        <li><span style="font-weight: 400;">{{ omember }}</span></li>
```

Comment:
I am not sure that `<span style="font-weight: 400;">` is necessary. Does it change anything when you remove it? (see also below)